### DISCUSSION TITLE
Issue with github profile-page

### DISCUSSION THREAD

**Discussion by Sam44323** [community/community#31392] [Other Feature Feedback, Questions, & Ideas] [answered]
There seems to be a UI issue with the profile page for the timing part... It's not coming up all the time but surely been observing this for quite some time!

PFA a snapshot!
<img width="431" alt="Screenshot 2022-08-30 at 11 08 03 PM" src="https://user-images.githubusercontent.com/65643151/187505898-6e3fba7f-63aa-4a4e-b49d-7fe2f4753fc0.png">

**Comment by gabodmc** [comment]
I had the same issue.
It happens when you have active zoom in the browser.

**Comment by radar-blip** [comment, marked as answer]
Lots of people are seeing this issue, its been reported and should be fixed soon. 😃
